documentation/src/main/asciidoc/public/getting-started/orm/index.adoc

```diff
@@ -14,12 +14,12 @@
 :relfilesuffix: /../en-US/html_single/index.html
 
 [[mapper-orm-getting-started]]
-= [[getting-started]] Getting started with the Hibernate ORM integration
+= [[getting-started]] Getting started with Hibernate Search in Hibernate ORM
 
 :numbered:
 
 This guide will walk you through the initial steps required
-to index and query your Hibernate ORM entities using Hibernate Search.
+to index and query your http://hibernate.org/orm/documentation[Hibernate ORM] entities using http://hibernate.org/search/documentation[Hibernate Search].
```

Comment:
Good idea, but I'd just point to the homepage for each project. If this was only about documentation, we would point to the documentation directly.

```suggestion
to index and query your http://hibernate.org/orm/[Hibernate ORM] entities using http://hibernate.org/search/[Hibernate Search].
```